packages/lib/har/harUtils.spec.ts

```diff
@@ -118,7 +120,7 @@ describe('harUtils', () => {
         size: 25,
         encoding: 'base64',
         text: content,
-      });
+      }) as Buffer;
```

Comment:
```suggestion
      });
```
It is no longer needed to cast to `Buffer`.